### PR TITLE
google analtics for angular and beta

### PR DESCRIPTION
**Pull request by robguthrie**
not ready

### CODE REVIEW

File: app/views/layouts/angular.html.haml [hunk 1]
```diff
@@ -14,7 +25,7 @@
     %start_menu
 
 
-    - if Rails.env.production?
+    - if Rails.env.production? and ENV['ERRBIT_HOST']
```
Comment (gdpelican) [inline app/views/layouts/angular.html.haml]: Not sure I understand why we're hiding the turn off angular option if Errbit isn't set up?

Comment (robguthrie) [inline app/views/layouts/angular.html.haml]: good point

File: lib/tasks/loomio.rake [hunk 2]
```diff
@@ -1,4 +1,31 @@
 namespace :loomio do
+  task :deploy do
```
Comment (robguthrie) [inline lib/tasks/loomio.rake]: @gdpelican simple move of deploy to ruby code. working for deploy to loomio-clone

Comment (robguthrie) [inline lib/tasks/loomio.rake]: I'll probably move it to lib/tasks/deploy.rb

**Pull request merged**